_test/tests/inc/pageutils_resolve_mediaid.test.php

```diff
@@ -1,58 +0,0 @@
-<?php
```

Comment:
Media resolver has no own tests in the File namespace because these were actual the same as for the pageresolver, but without startpages etc?

Reply:
correct